backend/src/main/java/com/redhat/cloud/notifications/recipients/rbac/AuthRequestFilter.java

```diff
@@ -0,0 +1,38 @@
+package com.redhat.cloud.notifications.recipients.rbac;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+import javax.ws.rs.client.ClientRequestContext;
+import javax.ws.rs.client.ClientRequestFilter;
+import java.io.IOException;
+import java.util.Base64;
+
+class AuthRequestFilter implements ClientRequestFilter {
+
+    String authInfo;
+
+    @ConfigProperty(name = "rbac.service-to-service.secret", defaultValue = "addme")
+    String secret;
+
+    @ConfigProperty(name = "rbac.service-to-service.application", defaultValue = "notifications")
+    String application;
+
+    AuthRequestFilter() {
+        String tmp = System.getProperty("rbac.service-to-service.exceptional.auth.info");
+        if (tmp != null && !tmp.isEmpty()) {
+            authInfo = Base64.getEncoder().encodeToString(tmp.getBytes());
```

Comment:
```suggestion
            authInfo = new String(Base64.getEncoder().encode(tmp.getBytes(UTF_8)), UTF_8);
```
Requires: `import static java.nio.charset.StandardCharsets.UTF_8;`